packages/@sanity/form-builder/src/inputs/PortableText/object/InlineObject.tsx

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable react/prop-types */
+import React, {FunctionComponent, useCallback, useMemo} from 'react'
+import {hues} from '@sanity/color'
+import {PortableTextChild, Type, RenderAttributes} from '@sanity/portable-text-editor'
+import {Marker, Path} from '@sanity/types'
+import {FOCUS_TERMINATOR} from '@sanity/util/paths'
+import styled, {css} from 'styled-components'
+import {Box, Card, Theme, Tooltip} from '@sanity/ui'
+import Preview from '../../../Preview'
+import {Markers} from '../../../legacyParts'
+import {RenderCustomMarkers} from '../types'
+
+type Props = {
```

Comment:
Prefer an interface named `InlineObjectProps` here:
```suggestion
interface InlineObjectProps {
```

Reply:
Se my other comment regarding this.